docs/pages/releases/2021-11-01.mdx

```diff
@@ -0,0 +1,112 @@
+import { Markdown, getStaticProps } from '../../components/Markdown';
+import { Emoji } from '../../components/primitives/Emoji';
+
+# Release: 1st November 2021
+
+Server-side Live Reloading is here! <Emoji symbol="🚀" alt="Rocket" /> Plus more updates as we finalise the Keystone 6 GA release.
+
+```json
+"@keystone-next/keystone": "x.x.x",
+```
+## Live Reloading <Emoji symbol="🚀" alt="Rocket" />
+
+Keystone now supports live reloading with `keystone-next dev`
+
+You can now update your GraphQL schema, change your hooks and access control, log errors and see how your data returns, then immediately use the playground to test it and iterate.
+
+This is in addition to the current support for live reloading changes to custom views in the Admin UI.
+
+### How it works
+
+When you run `keystone-next dev` now, it will start watching for changes to your config and schema. When changes are detected, Keystone will reload your config and schema and hot-swap the GraphQL endpoint.
+
+To balance performance and predictability, Keystone does not do a complete restart when changes are made. The things to know are:
+
+#### Prisma Schema Changes
+
+The Prisma Client is **not** reloaded as you make changes to the Keystone Schema. If you make changes that affect your Prisma schema, Keystone will exit the process and wait for you to run `keystone-next dev` again.
+
+This is because making database schema changes involves generating database migrations, and can result in data loss if those migrations are automatically run against your current database.
+
+When adding or removing lists and fields (except virtual fields), we recommend you finish making your changes then start the `dev` process again. This will generate a single migration for all the changes you have made, and interactively prompt you for how to deal with migrations that can't be run safely.
```

Comment:
```suggestion
When adding or removing lists and fields (except virtual fields), we recommend you finish making your changes then start the `dev` process again. This will generate a single migration for all the changes you’ve made, and interactively prompt you to resolve any migrations that can't be run safely.
```